src/states/elder/mod.rs

```diff
@@ -664,6 +665,7 @@ impl Elder {
 
         let src = Authority::PrefixSection(*trimmed_info.first_info.prefix());
         let content = MessageContent::NodeApproval(trimmed_info);
+
```

Comment:
Why the change?

Reply:
same as above, changed when resolve conflicts due to rebase. Will change it back.